addons/advanced_fatigue/XEH_postInit.sqf

```diff
@@ -12,10 +12,11 @@ if (!hasInterface) exitWith {};
     GVAR(ppeBlackout) ppEffectCommit 0.4;
 
     // - GVAR updating and initialization -----------------------------------------
-    if !(isNull ACE_player) then {
-        [ACE_player, objNull] call FUNC(handlePlayerChanged);
-    };
-    ["unit", FUNC(handlePlayerChanged)] call CBA_fnc_addPlayerEventHandler;
+    ["unit", FUNC(handlePlayerChanged), true] call CBA_fnc_addPlayerEventHandler;
```

Comment:
The `handlePlayerChanged` function needs to run at least once. If the unit EH triggered already, this will completly break AF. Would recommend to just revert this to how it was.